src/classes/ADV_Adapter.cls

```diff
@@ -0,0 +1,126 @@
+/*
+    Copyright (c) 2017, Salesforce.org
```

Comment:
@weizsforg The year in the copyright and apexdoc comment below be 2018 for new classes/triggers.

Reply:
Oh, Yes. I will update it.

Reply:
I see the change for the year in the apex doc comment, but the copyright year is still 2017 on the classes.